topics/chip-seq/tutorials/formation_of_super-structures_on_xi/tutorial.md

```diff
@@ -0,0 +1,862 @@
+> >         ------>
+> >           ------>
+> >
+> >     ----------------------------- [fragment]
+> >
+> >     ------>         <------
+> >       ------>         <------
+> >         ------>         <------
+> >           ------>         <------
+> > ```
+> > 
+> > The paired-end sequencing generates then 2 files:
+> > - One file with the sequences corresponding to foward orientation of all the fragments
+> > - One file with the sequences corresponding to reverse orientation of all the fragments
+> >
+> > Here `wt_H3K4me3_read1` corresponds to the forward reads and `wt_H3K4me3_read2` to the reverse reads.
+> {: .solution }
+{: .question}
+
+> ### {% icon hands_on %} Hands-on: Quality control
+>
+> 1. Run **Trim Galore!** {% icon tool %} with the following parameters
+>    - *"Is this library paired- or single-end?"*: `Paired-end`
+>       - {% icon param-file %} *"Reads in FASTQ format"*: `wt_H3K4me3_read1` (Input dataset)
+>       - {% icon param-file %} *"Reads in FASTQ format"*: `wt_H3K4me3_read2` (Input dataset)
+> 
+>       The order is important here!
+>
+>       > ### {% icon tip %} Tip: Not selectable files?
+>       >
+>       > If your FASTQ files cannot be selected, you might check whether their format is FASTQ with Sanger-scaled quality values (`fastqsanger`). You can edit the data type by clicking on the `pencil` symbol.
+>      {: .tip}
+>
+>    - *"Trim Galore! advanced settings"*: `Full parameter list`
+>       - *"Trim low-quality ends from reads in addition to adapter removal"*: `15`
+>       - *"Overlap with adapter sequence required to trim a sequence"*: `3`
+>       - *"Generate a report file"*: `Yes`
+>
+> 2. Inspect the generated txt file (`report file`)
+>
+>    > ### {% icon question %} Questions
+>    >
+>    > 1. In the forward reads, how many bp has been remove because of bad quality?
+>    > 2. And in the reverse reads?
+>    > 3. How many sequence pairs have been removed because at least one read was shorter than the length cutoff?
+>    >
+>    > > ### {% icon solution %} Solution
+>    > > 1. 32,198 bp (1.3%) (first `Quality-trimmed:`)
+>    > > 2. 116,414 bp (4.6%) (second `Quality-trimmed:`). It is not a surprise: we saw that the quality was dropping more at the end of the sequences for the reverse reads thant for the forward reads.
+>    > > 3. 1569 (3.14%) sequences (last line of the file)
+>    > {: .solution }
+>    {: .question}
+{: .hands_on}
+
+# Step 2: Mapping of the reads
+
+With ChiP sequencing, we obtain sequences corresponding to portion of DNA linked to histones with H3K4me3. As H3K4me3 opens the chromatime, nearby genes are more transcribed. It would be interesting to know if there is a difference in the quantity of DNA impacted by H3K4me3 and the impacted genes between active and inactive X chromosome.
+
+We first need to figure where the sequenced DNA fragments originated from in the genome. The short reads must be aligned to a reference genome and then identify the binding sites.
```

Comment:
figure out?

Reply:
Only reads, not short reads.
reference genome `to` identify the binding sites.?